libs/common/src/constants/tfm/team-ids.ts

```diff
@@ -1,7 +1,21 @@
+import { PdcTeamId, TeamId } from '../../types';
+
+export const ALL_PDC_TEAM_IDS = ['PDC_READ', 'PDC_RECONCILE'] as const;
+
+export const ALL_TEAM_IDS = [
+  'UNDERWRITING_SUPPORT',
```

Comment:
could we not have a seperate array for ALL_UNDERWRITING_IDS? like the PDC one

Reply:
it's not currently used (and if we have it, we'd want to make a type of it etc etc) -- So I'm verging on yagni until we have a use for this variable to avoid any potential issues this causes